app.py

```diff
@@ -16,15 +23,64 @@
   "required": ["firstname"]
 }
 
+schema2 = {
+  "type": "object",
+  "properties": {
+    "path": { "type": "string" }
+  },
+  "required": ["path"]
+}
+
+#homepage
 @app.route('/')
 def hello_world():
     return 'Hello, World!'
     
+#take json input (according to schema) and return json
 @app.route("/json", methods=['POST'])
 @expects_json(schema)
 def read_json():
     timestamp = datetime.datetime.now().isoformat()
     if 'lastname' in request.json:
         return jsonify({"greeting": "Hello, " + request.json['firstname']+" "+request.json['lastname'], "timestamp":timestamp})
     else:
-        return jsonify({"greeting": "Hello, " + request.json['firstname'], "timestamp":timestamp})
\ No newline at end of file
+        return jsonify({"greeting": "Hello, " + request.json['firstname'], "timestamp":timestamp})
+    #curl -H "Content-Type: application/json" --data '{"firstname": "Marie","lastname":"Gronemeyer","age":29}' http://127.0.0.1:5000/json
+
+#run some jobs in a rq
+@app.route("/rq")
+def try_rq():
+  q = Queue(connection=redis)
+  result = q.enqueue(add, 5,5)
+  result2 = q.enqueue(count_lines, 'poppunk/input_data/qfile.txt')
+  print(result.id)
+  return jsonify([{"id": result.id, "status":"curl http://127.0.0.1:5000/status/"+result.id,"result": "curl http://127.0.0.1:5000/result/"+result.id},
```

Comment:
I really like that the resource urls are returned in the response. But I would return just the url, minus the `curl`, as these will be more machine readable if they're ever programmed against.
```suggestion
  return jsonify([{"id": result.id, "status":"http://127.0.0.1:5000/status/"+result.id,"result": "http://127.0.0.1:5000/result/"+result.id},
```